_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -70662,7 +70662,7 @@
 "cvh" = (
 /obj/item/radio/beacon,
 /obj/effect/decal/warning_stripes/red/hollow,
-/turf/open/floor/plasteel/airless/ibombtest,
+/turf/open/floor/plasteel/airless/bombtest,
```

Comment:
This comment was TOTALLY in case of dataloss. No one someone accidentally put an 'i' here. Ha! How would that even happen! 😰

Reply:
I legitimately do not know why this turf even exists.